.github/workflows/test_linux.yaml

```diff
@@ -0,0 +1,45 @@
+name: Linux CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - nightly
```

Comment:
Is is possible to merge the workflows for windows + linux by putting them in the matrix?

Or is there any advantage to keeping them separate?